### PR TITLE
Chat page revamp

### PR DESCRIPTION
**Pull request by CastyiGlitchxz**
I really don't know what to put here.

This is a pull request for the new user interface for unnamed chat.

### CODE REVIEW

File: static/styles.js
Comment (cserver45) [inline static/styles.js:137]: for the values that have names in css, maybe we can change them to those names (ex: line 137 would change to `white`)

it's just a suggestion

File: static/styles.js
Comment (cserver45) [inline static/styles.js:143]: and here would be `black`

File: static/styles.js
Comment (cserver45) [inline static/styles.js:179]: line 179 should be expanded out to `#111111`

File: static/styles.js
Comment (cserver45) [inline static/styles.js:106]: expand out to `#ffffff`, or in this case, maybe change to `black`

File: static/styles.js
Comment (cserver45) [inline static/styles.js:75]: add the rest of the comments here (they are defined in `lightTheme` I think?)

File: static/styles.js
Comment (cserver45) [inline static/styles.js:130]: add the rest of the comments here (they are defined in `lightTheme` I think?)

File: static/styles.js
Comment (cserver45) [inline static/styles.js:171]: add the rest of the comments here (they are defined in `lightTheme` I think?)

**Review by cserver45 — CHANGES_REQUESTED** [review]
Some things I want changed for future readability, but others I want feedback from @01000011-00110111 and you first

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:130]: Completed

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:171]: Completed

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:106]: So I expanded it out to ```#ffffff```, because it blends in with the rest of the code better than ```black```

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:179]: Completed

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:143]: I kept it as ```#000000```, because it blends in with the rest of the code better than ```black```

File: static/styles.js
Comment (CastyiGlitchxz) [reply static/styles.js:137]: I kept the CSS the same because it all blends in together.

Comment (cserver45) [reply static/styles.js:137]: understandable

**Comment by cserver45** [comment]
did you forget to push to the repo? I don't see the changes you did

**Comment by CastyiGlitchxz** [comment]
My apologies, I just pushed it to repo.

**Review by cserver45 — APPROVED** [review]
I approve, wait for @01000011-00110111 approval first before merging

**Comment by CastyiGlitchxz** [comment]
Thanks for the approval and I think I know what is conflicting with main.

**Review by 01000011-00110111 — CHANGES_REQUESTED** [review]
get those merge conflicts fixed first

**Review by 01000011-00110111 — APPROVED** [review]
Love the work merge it in.

**Comment by 01000011-00110111** [comment]
Make sure to close all issues when merged

**Comment by cserver45** [comment]
merged

**Pull request merged**
